### PR TITLE
Guide to submit jobs using CueSubmit

### PR DESCRIPTION
**Pull request by sharifsalah**
An initial version of a guide to submitting jobs using CueSubmit.

I'll follow up with a PR to try adding the video guide to this page as well.

Staged at:

https://5c8bbc17a943140008232c60--elated-haibt-1b47ff.netlify.com/docs/user-guides/submitting-jobs/

### CODE REVIEW

File: content/en/docs/User guides/submitting-jobs.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+1.  Select a **Show**.
+
+    A show is a group of related jobs to be processsed. Jobs you submit to
+    OpenCue exist within a show.
+
+1.  Enter a **Shot**, which is a series of uninterupted frames you need to
+    render.
+
+1.  Enter a **Layer Name** to name the first layer in your job.
```
Comment (bcipriano) [inline content/en/docs/User guides/submitting-jobs.md]: For shot and layer name - one thing I noticed this past week demoing OpenCue to potential users is that they're confused about what values they should enter here.

Maybe we should specify that they are arbitrary strings? I.e. "shot" should be the name of the shot but _can_ be anything you want? And layer name can be whatever you want, but you should pick something descriptive based on what the layer is doing, such as "render" or "compositing".

File: content/en/docs/User guides/submitting-jobs.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    {{% alert title="Note" color="info"%}}Layer names must contain more than
+    3 characters and contain no spaces.{{% /alert %}}
+
+1.  In **Command to Run**, enter the shell command you want to run to render
+    the layer.
+
+    The command depends on the rendering software you're running.
```
Comment (bcipriano) [inline content/en/docs/User guides/submitting-jobs.md]: We should also add a bit about how to use the frame number placeholder.

https://cloud.google.com/solutions/creating-a-render-farm-on-gcp-using-opencue#submit_a_job has an example using `#IFRAME#`.

File: content/en/docs/User guides/submitting-jobs.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+1.  In **Frame Spec**, enter the range of frames you want to process.
+
+    A frame spec consists of a start time, an optional end time, a step,
+    and an interleave. To add multiple ranges together, separate them
+    by commas. For detailed examples of the frame spec syntax, click **?**.
+
+1.  Set **Job Type** to either **Shell** or the name of the software package
+    you're using for this layer in the job.
+
+1.  Select the required **Services** from the available list for your job.
```
Comment (bcipriano) [inline content/en/docs/User guides/submitting-jobs.md]: You might mention that services are used to match jobs with machines.

File: content/en/docs/User guides/submitting-jobs.md (hunk 1 above)
Comment (sharifsalah) [reply content/en/docs/User guides/submitting-jobs.md]: Great feedback, thanks. I was starting to think the same thing as I've been testing OpenCue myself more rigorously to prepare to demo it.

I've updated the doc.

File: content/en/docs/User guides/submitting-jobs.md (hunk 2 above)
Comment (sharifsalah) [reply content/en/docs/User guides/submitting-jobs.md]: Thanks and done. I've also expanded the explanation of 'Command to Run' with a brief example.

File: content/en/docs/User guides/submitting-jobs.md (hunk 3 above)
Comment (sharifsalah) [reply content/en/docs/User guides/submitting-jobs.md]: Thanks and done.

**Comment by sharifsalah** [comment]
Thanks Brian, I was hoping to add a few screenshots but wanted to wait for initial thoughts and feedback so thank you for your comments. On the whole, it's best to avoid too many screenshots as they add a lot of maintenance overhead but I think a few are warranted for CueSubmit and CueGUI. The updated doc is staged at:

https://5cb5f4707dfa6f0008ad77d0--elated-haibt-1b47ff.netlify.com/docs/user-guides/submitting-jobs/

Let me know what you think.

**Comment by sharifsalah** [comment]
Agree that it needs a lot more explanation. I was thinking an end-to-end tutorial to illustrate submitting multiple related jobs would be a good way to teach the concepts and demonstrate ow the various tools relate to each other.

**Pull request merged**